pkg/utils/utils.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+package utils
```

Comment:
How about changing the name of this package to `mocks`?

Reply:
I can do this.